ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py

```diff
@@ -0,0 +1,40 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+from hypothesis import strategies as st
+
+
+# asanyarray
+@handle_frontend_test(
+    fn_tree="numpy.asanyarray",
+    dtype=helpers.get_dtypes("valid", full=False),
+    order=st.sampled_from(["C", "F", "A", "K"]),
+    like=helpers.get_dtypes("valid", full=False),
+)
+def test_numpy_asanyarray(
+    a,
```

Comment:
The input `a` still needs to be generated in the decorator. It should be "Input data, in any form that can be converted to an array. This includes scalars, lists, lists of tuples, tuples, tuples of tuples, tuples of lists, and ndarrays.". The input `like` should also be generated in the same way.

Reply:
I though I did that of input `like`.  I hope I succeed this time. Thanks for your patience.